### DISCUSSION TITLE
What's the plan with online deal making?

### DISCUSSION THREAD

**Discussion by kernelogic** [filecoin-project/lotus#5657] [General Q&A] [answered]
I know for slingshot the majority of deals have gone offline due to high fail rate of online data transfer. Has there been any progress or initiative to fix these?

**Comment by dirkmc** [comment, marked as answer]
We are continuing to work on improving deal success, and in particular making improvements to data transfer. This is one of our highest priority projects. Recently we did some reorganization in order to dedicate more resources to it, we're still working out all the details but for the time being you can follow current progress on this milestone:
https://github.com/filecoin-project/lotus/milestone/7

And if you're interested in the details take a look at the go-fil-markets and go-data-transfer projects:
https://github.com/filecoin-project/go-fil-markets/pulls
https://github.com/filecoin-project/go-data-transfer
